pkg/trtl/server.go

```diff
@@ -162,6 +179,14 @@ func (t *Server) Shutdown() (err error) {
 	return nil
 }
 
+//===========================================================================
+// Initialize Prometheus metrics
+//===========================================================================
+
+func init() {
+	initMetrics()
```

Comment:
Should we init the metrics in `NewMetricsService`? I don't think we make any calls to the prometheus metrics until the server is started and running so we could probably wait until the Server is created. If you think it should remain in `init()` - could we move this to the top of this file; Because the `init` runs on import always, I like to have that at top of mind in the code so we don't forget what is being run in `init`.